peachjam/management/commands/backfill_timeline.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime, timedelta
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.db.models.functions import TruncDate
+
+from peachjam.models import TimelineEvent
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    help = "Backfill timeline events for users"
+
+    def handle(self, *args, **kwargs):
+        users = User.objects.filter(following__isnull=False).distinct()
+        total_users = users.count()
+        ten_days_ago = datetime.now() - timedelta(days=10)
+        self.stdout.write(f"Found {total_users} users with follows")
+
+        for user in users:
+            self.stdout.write(f"Processing user {user.username}...")
+            with transaction.atomic():
+
+                TimelineEvent.objects.filter(user_following__user=user).delete()
+                for follow in user.following.all():
```

Comment:
If you refactor UserFollowing a bit you can remove all this duplicate code and do something like this:

```
for follow in user.following.all():
  follow.last_alerted_at = ten_days_ago
  documents = follow.get_new_followed_documents
  follow.create_timeline_entries(documents)
```

That will help reduce bugs in duplicated code.